src/redux/categories/CategorySlice.js

```diff
@@ -0,0 +1,12 @@
+export const checkStatus = 'CHECK_STATUS';
```

Comment:
- [ ] Same here, kindly follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux)